telegram/_message.py

```diff
@@ -572,6 +572,17 @@ def chat_id(self) -> int:
         """:obj:`int`: Shortcut for :attr:`telegram.Chat.id` for :attr:`chat`."""
         return self.chat.id
 
+    @property
+    def id(self) -> int:  # pylint: disable=invalid-name
+        """
+        .. versionchanged:: 20.0
+          :obj: `int` : Convenience property. This is to reduce errors when users
+          use Message.id instead of Message.message_id which is inconsistent with most
+          of the other properties which follow a similar pattern.
+
```

Comment:
no need to mention the reason for this addition imo. We can just have something like:

```suggestion
        :obj:`int`: Shortcut for :attr:`message_id`.
        
            .. versionadded:: 20.0
```

Reply:
That's great! I was having a bit of an issue with that.